src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/RSACng.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace System.Security.Cryptography
+{
+    public sealed class RSACng : RSA
+    {
+        public RSACng()
+        {
+            throw new NotImplementedException(SR.WorkInProgress);
```

Comment:
I already cross-built this on the K branch to check this out - Open-source assemblies don't trigger the NI cop.